scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -238,6 +238,9 @@ trait TypedPipe[+T] extends Serializable {
   /** adds a description to the pipe */
   def withDescription(description: String): TypedPipe[T] = new WithDescriptionTypedPipe[T](this, description)
 
+  /* add a config parameter to this pipe */
+  def withExtraConfig(key: String, value: String): TypedPipe[T] = new WithExtraConfig[T](this, key, value)
```

Comment:
You mean instead of:

    def withExtraConfig(key: String, value: String): TypedPipe[T]

something like this:

    def withExtraConfig(extraConf: Config): TypedPipe[T]